main.js

```diff
@@ -0,0 +1,205 @@
+const xBttn = document.getElementById('mobileMenuIndx');
+const mobMenuIndex = document.getElementById('mobMenuIndex');
+const xMenuBttn = document.getElementById('xMenuBttn');
+const mainProgram = document.getElementById('mainProgramLink');
+const featuredSpeakers = document.getElementById('featuredLink');
+
+function openMobMenu() {
+  mobMenuIndex.classList.add('lr-menu');
+  mobMenuIndex.style.visibility = 'visible';
+}
+
+function closeMobMenu() {
+  mobMenuIndex.style.visibility = 'hidden';
+  mobMenuIndex.classList.remove('lr-menu');
+}
+
+xBttn.addEventListener('click', openMobMenu);
+xMenuBttn.addEventListener('click', closeMobMenu);
+mainProgram.addEventListener('click', closeMobMenu);
+featuredSpeakers.addEventListener('click', closeMobMenu);
+
+const speakersInfo = [
+  {
+    deskDisp: 'flex',
+    mobDisp: 'flex',
+    id: 'speaker1',
+    image: 'images/speaker1.png',
+    speakerName: 'Yochai Benkler',
+    resume: 'Berkham Professor of Environmental Legal Studies at Harvard Law School',
+    info: 'Benkler studies commons-based peer production, and published his seminal book The Wealth of Forests in 2007',
+  },
+  {
+    deskDisp: 'flex',
+    mobDisp: 'flex',
+    id: 'speaker2',
+    image: 'images/speaker2.png',
+    speakerName: 'SohYeong Noh',
+    resume: 'Director of BioArt Centre and a Board member of CC Korea',
+    info: 'As the main venue for bioart production in Korea, Nabi promotes cross-disciplinary collaboration',
+  },
+  {
+    deskDisp: 'flex',
+    mobDisp: 'none',
+    id: 'speaker3',
+    image: 'images/speaker3.png',
+    speakerName: 'Lila Tretikov',
+    resume: 'Executive Director of the Wikimedia Foundation',
+    info: 'Lorem three ipsum dolor sit amet consectetur adipisicing elepellendus ea vitae earum!',
```

Comment:
I noticed some of your content in this section is overlapping please fix this issue. You can achieve this by adding margins and padding. 

This is your design: 

 
![featurd speakers ](https://user-images.githubusercontent.com/79506105/192028938-d7821064-eaa4-4d5d-8634-5ebb6654db5f.png)
 
Your design should look like this:

![Screen Shot 2022-09-23 at 20 01 59](https://user-images.githubusercontent.com/79506105/192029393-d2bc1f9c-5396-4f0b-b37d-422da4f920c3.png)

And please add a white back ground color